src/vfxt/src/createUiDefinition.json

```diff
@@ -65,7 +65,7 @@
           "type": "Microsoft.Common.TextBlock",
           "visible": true,
           "options": {
-            "text": "Please provide the AAD role GUID used to create the Avere vFXT.",
+            "text": "Provide the GUID for the access control role to use to create the Avere vFXT cluster.",
```

Comment:
As of today (1/16), this is correct. It might not be correct by Monday. We may be able to remove this field.